src/axolotl/utils/callbacks/__init__.py

```diff
@@ -748,6 +748,9 @@ def on_train_begin(
                     mode="w", delete=False, suffix=".yml", prefix="axolotl_config_"
                 ) as temp_file:
                     copyfile(self.axolotl_config_path, temp_file.name)
+                    at = wandb.Artifact(f"config-{wandb.run.id}", type="axolotl-config")
+                    at.add_file(temp_file.name)
+                    wandb.log_artifact(at)
```

Comment:
```suggestion
                    artifact = wandb.Artifact(f"config-{wandb.run.id}", type="axolotl-config")
                    artifact.add_file(temp_file.name)
                    wandb.log_artifact(artifact)
```
the linter doesn't like `at`